storage/src/main/java/tech/pegasys/teku/storage/store/Store.java

```diff
@@ -521,10 +524,11 @@ UInt64 getHighestVotedValidatorIndex() {
     }
   }
 
-  VoteTracker getVote(UInt64 validatorIndex) {
+  Optional<VoteTracker> getVote(UInt64 validatorIndex) {
     readLock.lock();
     try {
-      return votes.get(validatorIndex);
+      if (validatorIndex.intValue() >= votes.length) return Optional.empty();
```

Comment:
nit: In our coding style we always use braces
```suggestion
      if (validatorIndex.intValue() >= votes.length) {
        return Optional.empty();
      }
```

Reply:
done